sips/sip-010/sip-010-fungible-token-standard.md

```diff
@@ -0,0 +1,132 @@
+## Preamble
```

Comment:
Formatting concern -- for consistency with the other SIPs, can you make each section header H1?  So, `# Preamble` instead of `## Preamble`.